subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/SimpleInteractionContext.kt

```diff
@@ -0,0 +1,72 @@
+package com.avito.android.test
+
+import android.view.View
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Root
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.ViewAssertion
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.matcher.ViewMatchers
+import com.avito.android.test.interceptor.ActionInterceptor
+import com.avito.android.test.interceptor.AssertionInterceptor
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+
+open class SimpleInteractionContext(
+    private val matcher: Matcher<View>,
+    private val rootMatcher: Matcher<Root>? = null,
```

Comment:
The key change.
Now can override root matcher. 
Example: `DialogInteractionContext`